Sources/OpenAPIRuntime/URICoder/Decoding/URIValueFromNodeDecoder.swift

```diff
@@ -345,16 +344,19 @@ extension URIValueFromNodeDecoder {
     /// contains a value for the provided key.
     /// - Parameter key: The key for which to look for a value.
     /// - Returns: `true` if a value was found, `false` otherwise.
-    /// - Throws: When parsing the root fails.
-    func containsElementInCurrentDictionary(forKey key: String) throws -> Bool {
-        try withCurrentDictionaryElements { dictionary in dictionary[key[...]] != nil }
+    func containsElementInCurrentDictionary(forKey key: String) -> Bool {
+        do { return try withCurrentDictionaryElements { dictionary in dictionary[key[...]] != nil } } catch {
+            return false
+        }
```

Comment:
nit: I still think if we're just swallowing the error and returning something else, then this is the nicer pattern, but it's not a blocker:
```suggestion
        (try? withCurrentDictionaryElements { dictionary in dictionary[key[...]] != nil }) ?? false
```

Reply:
Updated